content/docs/welcome-guide/create/creating-projects-using-cli.md

```diff
@@ -1,59 +1,66 @@
 ---
 linktitle: Creating Projects Using the CLI
 title: Creating Projects Using the Command Line Interface
-description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the CLI.
+description: Learn how to create and build new Open 3D Engine (O3DE) projects from the default project template using the command line (CLI).
 weight: 200
 toc: true
 ---
 
-The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using the command line interface (CLI).
+The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using a command line interface (CLI).
```

Comment:
```suggestion
The tutorial on this page and the video below provide an introduction to project configuration and building in Open 3D Engine (O3DE) using the `o3de` command line interface (CLI).
```

Reply:
If we want to use a term such as `o3de` CLI, then I should also explain what that is, or link to a definition topic.